core/assumes/sat_checker.go

```diff
@@ -8,10 +8,6 @@ import (
 	"github.com/juju/errors"
 )
 
-// A link to a web page with additional information about features,
-// the Juju versions that support them etc.
-const featureDocsURL = "https://juju.is/docs/olm/supported-features"
```

Comment:
question: why remove the link to documentation?

Reply:
The error messages should be self-explanatory and not require external resources to understand. This was previously necessary because the error message was rather cryptic, and the user would have to consult the docs to find out what it meant.